<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificacion</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBox2">
     <property name="text">
      <string>Pagado</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Importe:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Numero Comprobante de Venta:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Nombre del Cliente:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btnCancelar">
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btnModificar2">
     <property name="text">
      <string>Moificar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="leCliente2"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="leImporte2"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="leNcv2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
